### PR TITLE
Clean up autopilot parametets

### PR DESCRIPTION
**Pull request by hidmic**
This pull requests cleans up `autopilot` parameters, and avoid duplicating those already defined in actuator driver nodes. Some additional clean up was necessary in `fin_control` too.

### CODE REVIEW

File: autopilot/src/autopilot.cpp [hunk 1]
```diff
@@ -88,7 +88,7 @@ AutoPilotNode::AutoPilotNode(ros::NodeHandle& node_handle) : nh(node_handle)
   nh.param<double>("/autopilot_node/depth_imin", depthIMin, 0.0);
   nh.param<double>("/autopilot_node/depth_d", depthDGain, 0.0);
 
-  nh.param<double>("/autopilot_node/max_ctrl_plane_angle", maxCtrlPlaneAngle, 10.0);  // degrees
+  nh.param<double>("/fin_control/max_ctrl_fin_angle", maxCtrlFinAngle, 10.0);  // degrees
   nh.param<double>("/autopilot_node/max_depth_command", maxDepthCommand, 10.0);       // degrees
```
Comment (hidmic) [inline autopilot/src/autopilot.cpp:92]: This maximum depth command seems to actually be a maximum pitch command (?).

**Comment by hidmic** [comment]
There's also an RPM per knot constant in here that's duplicated with the one in `pose_estimator` and that IMHO does not belong to any of the two (as it depends on thruster and vehicle construction). Perhaps a system wide parameter, set via `launch`?

**Review by patapufate — APPROVED** [review]
LGTM. (nit: Pls fix the title.)

**Comment by hidmic** [comment]
>  Pls fix the title

The title is accurate, despite this PR having stuff not strictly related. I'll rebase-merge to not lose commit history.

**Comment by hidmic** [comment]
Alright, commit messages are OK. Moving forward!

> There's also an RPM per knot constant in here that's duplicated with the one in pose_estimator and that IMHO does not belong to any of the two (as it depends on thruster and vehicle construction). Perhaps a system wide parameter, set via launch?

@ChrisScianna when you get the time, let me know what you think about that.

**Pull request merged**